implementations/elixir/ockam/ockam_healthcheck/lib/healthcheck/healthcheck.ex

```diff
@@ -136,6 +138,7 @@ defmodule Ockam.Healthcheck do
          {:worker_init, _worker,
           {:handler_error, reason, _message,
            {Ockam.Transport.TCP, :handle_transport_message, [[]]}}}} ->
+          # How is this defined?
```

Comment:
👀

Reply:
This pattern match was taken from the crash report basically.